src/block/progress-circle/save.js

```diff
@@ -0,0 +1,53 @@
+import {
+	BlockDiv,
+	CustomCSS,
+	getResponsiveClasses,
+} from '~stackable/block-components'
+import classnames from 'classnames'
+
+import { InnerBlocks } from '@wordpress/block-editor'
+
+const Save = ( { className, attributes } ) => {
+	const responsiveClass = getResponsiveClasses( attributes )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-progress-circle',
+		responsiveClass,
+	] )
+
+	return (
+		<BlockDiv.Content
+			className={ blockClassNames }
+			attributes={ attributes }
+		>
+			<CustomCSS.Content attributes={ attributes } />
+			<div
+				className="stk-progress-circle"
+				style={ {
+					'--thickness': `${ attributes.thickness }px`,
+					'--percent': attributes.percentage,
+					'--progress-color': attributes.progressColor,
+					'--progress-background': attributes.progressBackgroundColor,
+				} }
```

Comment:
I'm inclined to do these in `style.js` instead of in `save.js` so that if we need to change things because of an update, we won't have to deprecate things.